assets/component-price.css

```diff
@@ -64,10 +64,15 @@
 }
 
 .price--sold-out .price__badge-sold-out,
-.price--on-sale .price__badge-sale {
+.price--on-sale .price__badge-sale,
+.volume-pricing--sale-badge .price__badge-sale {
   display: inline-block;
 }
 
+.volume-pricing--sale-badge .price__badge-sale {
+  margin-left: 0.5rem;
+}
+
 .price--on-sale .price__sale {
```

Comment:
We show compare at in cart and cart drawer https://screenshot.click/18-01-j2ene-qju7h.png

Should we show it there too? https://screenshot.click/18-01-j2ene-qju7h.png

https://screenshot.click/18-02-bvw2h-nn6y8.png

Reply:
We do not show compare at on cart and cart drawer for D2C. What you have on your screenshot is the discount applied to that product. ( A bit confusing, I know)

Video: https://screenshot.click/18-20-gpkvx-pc5di.mp4

So, not let's not show the compare at price on cart.